### ISSUE TITLE
2.1.5 does not build with gecode 5.1.0

### ISSUE THREAD

**Issue by stefanoteso** [MiniZinc/libminizinc#167]
Hi,

I've tried compiling a fresh git clone of 2.1.5 together with gecode 5.1.0 -- with no success. I've used the BUILD.sh script (adjusting the paths as needed).

The first build issues is due to the fact that Gecode::IntConLevel was renamed to IntPropLevel in gecode 5.0 [1]. libminizinc still expects IntConLevel to exist:

  https://github.com/MiniZinc/libminizinc/blob/master/solvers/gecode/gecode_solverinstance.cpp#L853

After fixing that, compilation still fails on gecode_solverinstance.cpp, see log at:

    https://paste.fedoraproject.org/paste/wU276WxcMY9vREk-xNo3Uw

Am I doing something wrong? Is gecode 5.* supported?

BTW, this is on gcc 6.3.0 an a linux machine.

Thanks!

[1] http://www.gecode.org/doc-latest/reference/PageChange.html#SectionChanges_5_0_0

**Comment by guidotack** [comment]
The Gecode interface inside libminizinc is currently lagging behind a bit. Please compile without Gecode for the time being, and use the Flatzinc interface that is shipped as part of Gecode.

**Comment by stefanoteso** [comment]
Indeed, building without the gecode plugin works like a charm. Thanks.

**Comment by skosch** [comment]
Do you have a rough idea when this will get updated (even just in develop)? I would love to try running some CP problems via the JSON interface, and having either Gecode or OR-tools available would be amazing.

**Comment by guidotack** [comment]
Later this year. For the time being you can use the mzn-fzn driver with the regular fzn-gecode (e.g. the one that's bundled with the MiniZinc IDE), it fully supports JSON.
